README.md

```diff
@@ -18,37 +18,47 @@ DOCKER_COMPOSE=docker-compose -f local-docker-compose.yml
 
 - install [Docker](http://docker.io)
 - bring in ICU source
-
-     cd src
-     git clone --branch master --depth 1 https://github.com/unicode-org/icu.git
-     cd icu
-     git lfs fetch
-     git lfs checkout
-     cd ../..
-
+````
+    cd src
+    git clone --branch master --depth 1 https://github.com/unicode-org/icu.git
+    cd icu
+    git lfs fetch
+    git lfs checkout
+    cd ../..
+````
 - run tests
-
+````
      make check-all
-
+````
 - build binaries
+````
+    make dist
+````         
+- sort into dist/icu4c-*/*
 
-     make dist
-     # sort into dist/icu4c-66.1/*
+The files should include the version label such as "69rc" for the release candidate of ICU version 69. The general availability for that woud be "69.1". 
+````
      ./sort-out-dist.sh
      ls -l dist/icu4c-*
-
+````
 - optional: symlink `src/` to `/src` on your system
 
-this will make error messages work
-
+This will make error messages work
+````
      sudo ln -sv `pwd`/{src,dist} /
+````
 
-- do some command line builds
-
+- Perform some command line builds to verify the release.
 ```
     $ docker-compose run ubuntu bash
+    # This creates a temporary docker shell with a name such as 'build@671f33b0fe95:~'
```

Comment:
```suggestion
    # This creates a temporary docker shell with a name such as 'build@59b67f6c5058:~'
```
kind of a nit, but would it be more clear to use the same docker ID as in the example commands below?
(Or is the ID intentionally different perhaps...?)

Reply:
Sure, that makes sense.